messages/3.0.0.txt

```diff
@@ -0,0 +1,5 @@
+# Breaking change
+
+This server is now based on the Vue Language Server 2.x rather than the deprecated and no longer updated Vetur.
+
+The legacy, deprecated Vetur is now available through the LSP-vetur package for those who really need it.
```

Comment:
Maybe extend the release notes with a guide on how the users should handle this breaking change.
For example, if they have a sublime-project file they would probably need to change the name from LSP-volar to LSP-vue.

Reply:
Keep in mind that this message will show up for existing LSP-vue and not LSP-volar users. There is not much the LSP-vue user can do in terms of migration - either just start using new server or switch to LSP-vetur.

Reply:
Right, I got confused by the diff